docs/ca/dev/best-practices/jenkins-operational-best-practices.md

```diff
@@ -33,7 +33,7 @@ The following diagram showcases different memory constraints you should consider
 
 ![memory-constraints](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/tutorials-and-howtos/howtos/jenkins-stability-checklist/memory_constraints.png)
 
-Keep in mind that each Jenkins executor can run one PHP job, which may potentially spawn multiple PHP threads or child processes. Each executor can consume RAM up to `memory_limit` value. 
+Keep in mind that each Jenkins executor can run one PHP job, which may potentially spawn multiple PHP threads or child processes. Each php thread can consume RAM up to `memory_limit` value. 
```

Comment:
technically those are not a traditional threads, but just a simple child processes

Reply:
updated to "processes"